### DISCUSSION TITLE
Screen Whitening Issue When Using HDR in RustDesk

### DISCUSSION THREAD

**Discussion by pp-X** [rustdesk/rustdesk#9952] [General]
### Bug Description

When using RustDesk for remote connection, enabling HDR (High Dynamic Range) results in the remote screen appearing significantly whitened, affecting usability.
![51fbb79eb1c4904f26710e2a7dd9de6](https://github.com/user-attachments/assets/e3d962c7-65a2-403d-b743-3822ce0d4904)

### How to Reproduce

“First, enable the HDR option on the computer. Then, connect using the client, and the issue can be reproduced. When I turn off HDR, the display returns to normal. You can toggle HDR on and off using the shortcut Win + Alt + B.”
![image](https://github.com/user-attachments/assets/308538b1-7e02-470d-bf17-6f8258120100)

### Expected Behavior

“I would like to suggest that when connecting through the client, the software should automatically detect if the display supports HDR and make necessary adjustments to accommodate HDR. This would ensure optimal display compatibility and performance.”

### Operating system(s) on local (controlling) side and remote (controlled) side

win11

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.2

### Screenshots

![51fbb79eb1c4904f26710e2a7dd9de6](https://github.com/user-attachments/assets/779c1546-d07d-4f6a-acae-7b5c94db53d5)
![image](https://github.com/user-attachments/assets/308538b1-7e02-470d-bf17-6f8258120100)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/7652
